dts/common/nordic/nrf54h20.dtsi

```diff
@@ -987,6 +987,19 @@
 				status = "disabled";
 				interrupts = <470 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			exmif: spi@95000 {
```

Comment:
Please keep the nodes ordered by address and locate this node right after the `usbhs` one.

Reply:
Changed the order